README.md

```diff
@@ -134,6 +134,8 @@ Pay attention that adding route alias is not supported. That said you **can't**
 mount Engine, at: '/engine', as: :engine
 ````
 
+If you need to redefine devise routes set up env `DEVISE_ROUTES_REDEFINED` as true and define devise related routes at host app.
+
```

Comment:
I would add a simple example here.

Reply:
example depends on host app needs, don't think we need to repeat devise documentation.

Reply:
added some examples, rebased with master and force pushed.